src/helpers/validators.js

```diff
@@ -74,4 +75,24 @@ export const isEthAddress = (addr = '') => {
 		return false
 	}
 	return true
+}
+
+export const isEthAddressZero = (addr) => {
+	return isEthAddress(addr)
+		? utils.bigNumberify(addr).isZero()
+		: false;
+}
+
+export const isEthAddressERC20 = async (addr) => {
```

Comment:
add this function and the abi to `services/smart-contracts/...`

Reply:
as a new validators.js in the folder or in the ethers.js file?

Reply:
`actions/erc20`